### ISSUE TITLE
Organizations search result component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#120]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to create parts of the search result component that is being made in #119. We want the features made in this issue to be conditionally switched to based on the whether `organization` is passed to the aforementioned component's props :)

The designs for this component can be found on the [Components tab in Figma](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=1606%3A585&t=5CR5mIoy4P0OjwjS-1) and can further be seen below:

![Mode=Light, Mobile=False, Type=Organization, Reduced=False](https://github.com/activist-org/activist/assets/24387426/91e93581-fcc8-4131-ad05-9f50d5ac2065)

Mobile:

![Mode=Light, Mobile=True, Type=Organization, Reduced=False](https://github.com/activist-org/activist/assets/24387426/65b1b5fb-a240-427f-b67f-c9a70a0beeda)

- This issue is blocked by #119 that makes the baseline component

### Contribution

Happy to support someone who wants to work on this or work on it myself once #119 is done :)

**Comment by andrewtavis** [comment]
Assigning myself to hold this issue for the team working on it for Code Night 🌙😊

**Comment by shreevaths1** [comment]
What needs to be done actually ?

**Comment by andrewtavis** [comment]
Hey @shreevaths1! Thanks for writing and your interest in activist. There’s already a PR for this issue in #144. You could potentially work on #121 or #122. Basically we’re adding parts to the component for a search result to style it according to if it’s an organization. For the other issues I mentioned we’re doing the same for events and resources.

**Comment by andrewtavis** [comment]
@james-jenkinson, just merged #144. FYI you're not in [the contributors](https://github.com/activist-org/activist/graphs/contributors) as of now, which normally is because the email that's for GitHub doesn't match the one that you used for Git :) The [email that you use for GitHub](https://github.com/settings/emails) should be the same that you have for git `config --global user.email` or locally for the repo :)

**Comment by andrewtavis** [comment]
Working on the minor changes we discussed in #144 now 🙃

**Comment by andrewtavis** [comment]
@james-jenkinson, dea9279 did some minor adjustments to add an alternate image, add some new data types as we discussed and I put the restricted string assignments `'organization' | 'event'`, etc where they could go :)

Closing this now and we can pick up future work in #121 and #122 🙌

**Comment by andrewtavis** [comment]
Let me know if you see anything that we should discuss in https://github.com/activist-org/activist/commit/dea9279c98994d7b8392f1552ab6319fdebe5169 though 😊
